### ISSUE TITLE
Support optionally inserting function arguments

### ISSUE THREAD

**Issue by nickbroon** [Sarcasm/company-irony#13]
The company-clang backend has `company-clang-insert-arguments` option that defaults to on.
A similar option for the irony backend would be great.

**Comment by Sarcasm** [comment]
What do you want exactly:

* to turn off the arguments expansion that we have right now with yasnippet?
* or you are not aware that with yasnippet enabled the arguments are expanded?

**Comment by nickbroon** [comment]
Thanks, I wasn't aware of that. How is the optional yasnippet support in company-irony enable?
It might be worth adding a note about this to the readme.

**Comment by Sarcasm** [comment]
Indeed, it's noted here: https://github.com/Sarcasm/irony-mode#elisp-dependencies

But it should totally appear in company-mode documentation too, I leave this issue open until it's done.

To have it enabled you just have to have yasnippet (`yas-minor-mode`) enabled in your buffer while the completion is triggered.

In my case I have installed yasnippet and enabled it via `yas-global-mode`, see: https://github.com/Sarcasm/.emacs.d/blob/bce936b2f63d56b785d0c91cec87923f6e3de9e9/sarcasm-elisp/sarcasm-packages/init-yasnippet.el#L24

**Comment by nickbroon** [comment]
Fantastic, thanks! Now working as I'd hoped.
